build.sbt

```diff
@@ -113,9 +113,11 @@ lazy val zincRoot: Project = (project in file(".")).
         uncommittedChanges.exists(_.nonEmpty)
       },
 
-      // Ignore "git.gitCurrentTags.value.isEmpty"
-      // i.e. allow not-on-tag commits to be considered isSnapshot := false
-      isSnapshot := git.gitUncommittedChanges.value,
```

Comment:
@dwijnand Given that scripted requires publish local, I would like to avoid commit SHA version when it's not tagged.